src/components/creator/CreatorPreview.tsx

```diff
@@ -46,44 +52,53 @@ const CreatorPreview = ({
     quickstartValues.setAllQuickStarts?.([quickStart]);
   }
 
-  if (
-    prevTask !== currentTask ||
-    quickstartValues?.activeQuickStartState === undefined
-  ) {
-    setPrevTask(currentTask);
+  if (prevStage !== currentStage) {
+    setPrevStage(currentStage);
 
-    if (currentTask !== null) {
-      quickstartValues.setQuickStartTaskNumber?.(
-        quickStart.metadata.name,
-        currentTask
-      );
-    } else {
+    if (currentStage.type === 'panel-overview') {
       quickstartValues.restartQuickStart?.(
         quickStart.metadata.name,
         quickStart.spec.tasks?.length ?? 0
       );
+    } else if (currentStage.type === 'task') {
+      quickstartValues.setQuickStartTaskNumber?.(
+        quickStart.metadata.name,
+        currentStage.index
+      );
     }
   }
 
   return (
-    <QuickStartContext.Provider value={quickstartValues}>
-      <QuickStartDrawer quickStarts={allQuickStarts}>
-        <section>
-          <Title headingLevel="h2" size="xl" className="pf-v5-u-mb-md">
-            Live card preview
-          </Title>
+    <Flex
```

Comment:
The preview should not have a white background.

![screenshot-stage_foo_redhat_com_1337-2024_07_17-10_20_53](https://github.com/user-attachments/assets/e053a008-6dd7-4fae-9e3b-809642497cab)

Reply:
It... doesn't on my end? On either Firefox or Chrome 
![image](https://github.com/user-attachments/assets/5ad54357-b084-42ca-88ba-060198a5612d)

Reply:
I use the latest Chrome. I did nothing special, the background is just white.

Reply:
Running it with `npm run build; BETA=true npm run static -- --port=8004` in learning-resources and `BETA=true CHROME_SERVICE=8000 CONFIG_PORT=8000 NAV_CONFIG=8000 LOCAL_APPS=learning-resources:8004~http npm run dev` in insights-chrome (on the latest master), I still get a grey background on Google Chrome.

The quickstart pulling focus in https://github.com/RedHatInsights/learning-resources/pull/57#pullrequestreview-2182278154 makes me think at least the quickstarts dependency is outdated (should be `5.4.0-prerelease.1`); could you double check that?

Reply:
Tested locally and I am not seeing the white background: 
![image](https://github.com/user-attachments/assets/72d65a0a-9e55-49b0-9d60-5282c0318469)